### PR TITLE
Add functional menu

### PR DESCRIPTION
**Pull request by MaevaWolff**
https://user-images.githubusercontent.com/56693082/126192542-adcd7b05-a9f4-4143-8754-e617eeb5a203.mov

### CODE REVIEW

File: src/components/Menu.tsx [hunk 1]
```diff
@@ -1,58 +1,90 @@
...
-const Header = styled.header`
-  border: 1px solid red;
+const Container = styled.header`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.primary};
   height: 100vh;
-  width: 7%;
-  transition: width 0.3s ease-in;
```
Comment (thomasliedts) [inline src/components/Menu.tsx:15]: Why do you remove the transition and the over ?

File: src/components/Menu.tsx [hunk 2]
```diff
@@ -1,58 +1,90 @@
...
+    path: "/urgency",
+    Icon: Urgency,
+    text: "Urgence",
+  },
+];
+
 export default function Menu() {
+  const location = useLocation();
+
+  const renderItemsOfMenu = itemsOfMenu.map(({ path, Icon, text }) => (
```
Comment (thomasliedts) [inline src/components/Menu.tsx:75]: Instead of create a const, you can place your map in the ul

File: src/components/Menu.tsx (hunk 1 above)
Comment (MaevaWolff) [reply src/components/Menu.tsx:15]: Our width should be fixed because the grid layout depends on this width

File: src/components/Menu.tsx (hunk 2 above)
Comment (MaevaWolff) [reply src/components/Menu.tsx:75]: Do you prefer to include this logic directly in the JSX?
I create a const for more readability

File: src/components/Menu.tsx (hunk 1 above)
Comment (thomasliedts) [reply src/components/Menu.tsx:15]: Ok !

File: src/components/Menu.tsx (hunk 2 above)
Comment (thomasliedts) [reply src/components/Menu.tsx:75]: Yes for the map you can include the logique in JSX

Comment (MaevaWolff) [reply src/components/Menu.tsx]: fixed 
fc643b6

**Review by Nstampfli — APPROVED** [review]

**Pull request merged**